### PR TITLE
DOCSP-38696 Add faq regarding fatal errors

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
## DESCRIPTION
Add faq regarding fatal errors.

## STAGING
https://preview-mongodbjocelynmendez1.gatsbyjs.io/cluster-sync/DOCSP-38696/faq/#what-should-i-do-if-mongosync-returns-a-fatal-error-

## JIRA
https://jira.mongodb.org/browse/DOCSP-38696

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6627d712b67dbf804a816d15

### CODE REVIEW

File: source/faq.txt [hunk 1]
```diff
@@ -136,6 +136,14 @@ These logs do not indicate that ``mongosync`` is failing or corrupting
 data. If a fatal error occurs, ``mongosync`` stops the sync and writes a
 fatal log entry. 
 
+What should I do if ``mongosync`` returns a fatal error?
```
Comment (jason-price-mongodb) [inline source/faq.txt:139]: Avoid monospace in headings

File: source/faq.txt [hunk 2]
```diff
@@ -136,6 +136,14 @@ These logs do not indicate that ``mongosync`` is failing or corrupting
...
+--------------------------------------------------------
+
+A fatal error indicates a problem that must be addressed and requires the
```
Comment (jason-price-mongodb) [inline source/faq.txt:142]: Rewrite to not use address, which has multiple meanings and is not global. Example, use "fixed" or "examined" or similar.

**Review by jason-price-mongodb — APPROVED** [review]
@jocelyn-mendez1 lgtm with suggestions. Cheers.

File: source/faq.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/faq.txt]: Updated.

File: source/faq.txt (hunk 2 above)
Comment (jocelyn-mendez1) [reply source/faq.txt]: Updated.

**Comment by jocelyn-mendez1** [comment]
Hi @edobranov, I have a updated the FAQ to include the fatal errors questions. Please let me know of any possible updates. Thanks in advance!

**Review by edobranov — APPROVED** [review]
LGTM!

**Pull request merged**